static-site/src/pages/groups.jsx

```diff
@@ -4,23 +4,39 @@ import { SmallSpacer, HugeSpacer, FlexCenter } from "../layouts/generalComponent
 import * as splashStyles from "../components/splash/styles";
 import { fetchAndParseJSON } from "../util/datasetsHelpers";
 import DatasetSelect from "../components/Datasets/dataset-select";
+import ListResources from "../components/ListResources/index";
 import { GroupCards } from "../components/splash/groupCards";
 import GenericPage from "../layouts/generic-page";
 import { UserContext } from "../layouts/userDataWrapper";
 import { DataFetchErrorParagraph } from "../components/splash/errorMessages";
 import { groupsTitle, GroupsAbstract } from "../../data/SiteConfig";
 
-const datasetColumns = ({isNarrative}) => [
-  {
-    name: isNarrative ? "Narrative" : "Dataset",
-    value: (d) => d.request.replace("/groups/", "").replace(/\//g, " / "),
-    url: (d) => d.url
-  },
-  {
-    name: "Group Name",
-    value: (d) => d.request.split("/")[2],
-    url: (d) => `/groups/${d.request.split("/")[2]}`
-  }
+const sourceId = "charon/groups";
+async function parserCallBackFxn(response) {
+    const datasets = (await response.json()).datasets;
```

Comment:
This uses `charon/getAvailable` but it wasn't super obvious to me - there's a big distance between this line and [where the use of charon is defined](https://github.com/nextstrain/nextstrain.org/blob/bc73c3b9336fb48602d918ef41452b69611abd55/static-site/src/components/ListResources/useDataFetch.ts#L35).

Suggestion: define the URL and make the fetch request directly in the definition of each `parserCallBackFxn`, rather than shared across all invocations of `useDataFetch` and conditions written inside of that. This would be a larger refactor, but I think could be in scope of this PR if the goal is to make ListResources re-usable outside of `/pathogens` and `/staging` pages.

Reply:
I structured this the way I did because it felt like the smallest effective change, and I wasn't sure we really wanted to effectively outsource the production of the ResourceListingInfo datastructure to the consumer of the <ListResources> component — which is where I think this suggestion ends up once the dust settles.

Anybody else have a strong opinion here?

Reply:
>  I wasn't sure we really wanted to effectively outsource the production of the ResourceListingInfo datastructure to the consumer of the component — which is where I think this suggestion ends up once the dust settles.

I'm not sure we need to (nor should) go this far. But I do think adding the `fetch` call to the callback (and therefore the API address) is a better separation of concerns.

Reply:
I refactored things so that the `<ListResources>` component gets passed a URL instead of a `sourceId`. I left the `fetch()` functionality where it is. With the way `useDataFetch` is currently implemented and structured, pulling the `fetch` call out into the resource listing call back would be fairly ugly, IMO.

The way things are structured now, `<ListResources>` fetches a URL and hands the response for that fetch off to the provided callback, which is responsible for producing an object with this shape: 

```json
{ 
  "pathPrefix": string,
  "pathVersions": (string|undefined)[],
}
```

I'm just not seeing where the `fetch` in the callback is an improvement over the updates I just pushed.

Reply:
> I'm just not seeing where the `fetch` in the callback is an improvement over the updates I just pushed.

The new updates are better, but I still see some benefits to having `fetch` in callback. Something like this, feel free to take or leave: c842d4d84c96af8eba3cb4c109f36e40c99ccca0